dev/PushNotifications/externs.h

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+#include "PushNotificationReceivedEventArgs.h"
+
+extern wil::unique_handle g_waitHandleForArgs;
+
+const winrt::hstring ACTIVATED_EVENT_ARGS_KEY = L"GlobalActivatedEventArgs";
```

Comment:
Should not have objects with nontrivial constructors as static data.

Also, shouldn't this be `inline`? Otherwise it seems like it would be an ODR violation.